lib/pages/chat/events/message_content.dart

```diff
@@ -126,14 +126,9 @@ class MessageContent extends StatelessWidget {
                 child: HtmlMessage(
                   event: event,
                   html: html,
-                  defaultTextStyle:
-                      Theme.of(context).textTheme.bodyLarge?.copyWith(
-                            // color: textColor,
-                            fontSize: bigEmotes ? fontSize * 3 : fontSize,
-                          ),
+                  defaultTextStyle: Theme.of(context).textTheme.bodyLarge,
                   linkStyle: TextStyle(
                     color: Theme.of(context).colorScheme.secondary,
-                    fontSize: bigEmotes ? fontSize * 3 : fontSize,
```

Comment:
are you sure links will have the same font size ? 
IMO it should be bodylarge too

Reply:
Sure

Reply:
<img width="1128" alt="Screenshot 2023-11-16 at 01 05 58" src="https://github.com/linagora/twake-on-matrix/assets/99852347/685cac57-d489-46ca-b26f-4bf8614911de">